analyzers/src/SonarAnalyzer.CSharp/SymbolicExecution/CSharp/CSharpExplodedGraph.cs

```diff
@@ -33,7 +33,7 @@
 using SonarAnalyzer.SymbolicExecution.Constraints;
 using SonarAnalyzer.SymbolicExecution.SymbolicValues;
 using StyleCop.Analyzers.Lightup;
-
+using ComparisonKind = SonarAnalyzer.SymbolicExecution.SymbolicValues.ComparisonKind;
```

Comment:
_Nitpick:_ Missing EOL after this `using` statement.
```suggestion
using ComparisonKind = SonarAnalyzer.SymbolicExecution.SymbolicValues.ComparisonKind;

```

Reply:
The `using` is gone, but the EOL is missing there - unintentionally removed by this PR.